### PR TITLE
Add scripts/update-dispensary-doc

### LINKED ISSUE #4030: Add a command to automatically update the list of (dispensary) libraries in the `README` file
We should not have to maintain the list of libraries manually. There is a JSON file (`libraries.json`) containing the URLs so we should parse this file and update the `README.md` file automatically. See this script from the `dispensary` project: https://github.com/mozilla/dispensary/blob/3edf6e910afb6fd24c1e10d9b0b52a48ad49634b/bin/build-doc

mentor: @willdurand

### PR DESCRIPTION
**Pull request by lucaseverett**
Fixes #4030

This PR adds bin/build-doc from the archived Dispensary repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -273,6 +277,8 @@ This is the (manual) process to update the "dispensary" libraries:
 
 Note: `hashes.txt` will be embedded into the addons-linter bundle.
 
+The `bin/build-doc` command updates the list of release pages in the `README.md` file based on the `src/libraries.json` file.
```
Comment (willdurand) [inline README.md:280]:
```suggestion
The `bin/build-doc` command updates the list of release pages above based on the `src/dispensary/libraries.json` file.
```

File: bin/build-doc [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
```
Comment (willdurand) [inline bin/build-doc:1]: Could you please move this file to `scripts/`? This is the folder that contains the non-production scripts in this repo. Also, I'd rename the script to something like `update-dispensary-doc`.

**Comment by lucaseverett** [comment]
@willdurand The file has been moved to the scripts folder, and the README file has been updated.

**Review by willdurand — APPROVED** [review]

**Comment by willdurand** [comment]
Excellent, thanks!

**Pull request merged**